From a9d2af7ba6ec8d2c43ff21089869ae835954c90f Mon Sep 17 00:00:00 2001
From: Bartek Grzybowski <b.grzybowski@partner.samsung.com>
Date: Fri, 4 Feb 2022 14:19:51 +0100
Subject: [PATCH] [MARIADB] Use common labels for the metrics Service selector

Setting the "prometheus: kube-prometheus" label selector for mariadb-galera
ServiceMonitor object is invalid since mariadb-galera-metrics service has
common labels assigned.

Without this change Prometheus (in kube-prometheus-stack) does not select
the mariadb-galera-metrics service for scraping and relevant target is
not created.

Change-Id: I64dfe83ff7fb448125f8726cf3ca33048ced04a8
Issue-ID: OOM-2925
Signed-off-by: Bartek Grzybowski <b.grzybowski@partner.samsung.com>
---
 kubernetes/common/mariadb-galera/values.yaml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/kubernetes/common/mariadb-galera/values.yaml b/kubernetes/common/mariadb-galera/values.yaml
index f49a89881..9f7c88213 100644
--- a/kubernetes/common/mariadb-galera/values.yaml
+++ b/kubernetes/common/mariadb-galera/values.yaml
@@ -608,8 +608,8 @@ metrics:
     ## ServiceMonitor selector labels
     ## ref: https://github.com/bitnami/charts/tree/master/bitnami/prometheus-operator#prometheus-configuration
     ##
-    selector:
-      prometheus: kube-prometheus
+    # selector:
+    #   prometheus: kube-prometheus

     ## RelabelConfigs to apply to samples before scraping
     ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#relabelconfig
--
2.17.1
